### ISSUE TITLE
Accessibility - Insufficient Size or Spacing ( Touch Targets)

### ISSUE THREAD

**Issue by outiswebsolutions** [outiswebsolutions/outis-web-solutions#17]
Abiding by WCAG ensuring proper size & spacing. 

Increase size of text for navigation links for navbar.
Increase space between navbar li elements.
Increase navbar height ensuring adequate space for the touch targets.

**Comment by outiswebsolutions** [comment]
Fixed by:
set padding for navbar to 1.5rem for touch targets
navbar img logo height 3rem & margin right to 1rem
navbar hyperlinks set to inline block display, set padding in rem for touch targets
added nav-links a css property  selector ensures proper padding for smaller screens.
